### PR TITLE
split up readme instructions for dev setup

### PR DESCRIPTION
**Pull request by pulkitsinghal**
@Bhushan001 - I will merge this within 16 hours since the changes have no code, only instructions.

### CODE REVIEW

**Comment by pulkitsinghal** [comment]
cc @kamal0808 - changes coming to README ... FYI

File: readme/local-ide-remote-docker.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+1. On local, Dropbox's autosync will create a directory and you can jump into it: `cd $SLAVE_WAREHOUSE_HOME`
+1. On local, open your favorite IDE and start working
+    * for example, visual studio can be opened with: `cd $SLAVE_WAREHOUSE_HOME && code .`
+1. On master, create empty env files: `cd $WAREHOUSE_HOME && touch .env worker.env worker2.env`
+    * run: `docker-compose run nodejs npm install inquirer --save-dev`
+    * then run the command `docker-compose run nodejs npm run generate-env -e=web` for generating `.env` for warehouse web service
+        * It will ask you for the environment variables to be configured.
+        * Once done, it will generate files for you as per the values you specified.
+    * then run the command `docker-compose run nodejs npm run generate-env -e=worker` for generating `worker.env` for warehouse worker service.
+    * `@Bhushan001` we need `worker2.env` as well
```
Comment (pulkitsinghal) [inline readme/local-ide-remote-docker.md]: @Bhushan001 - somethign to improve in the near future

**Comment by pulkitsinghal** [comment]
@bhushan and @harshadyeola - also pay attention to:
```
        # Once terraform creates queues, the appropriate
        # AWS_SQS_URL and AWS_SQS_REGION will be
        # automatically added to your .env and worker.env files.
```

that means your script should:
1. copy env example files
2. collect anything needed for `terraform.tfvars`
3. fill it in and kick off terraform
4. after terraform has update env files, your script should continue to collect only what's remaining without nuking the existing data or formatting in env files

Please create a new issue once you decide on an appropriate direction and afterwards also update the readme ofcourse.

**Comment by harshadyeola** [comment]
@pulkitsinghal instead of creating anoth readme dir you should have put these 2 files into `docs` directory already available in repository.

**Pull request merged**